### PR TITLE
Statistics: Added base statistical functions

### PR DESCRIPTION
**Pull request by albaEDA**
Improved statistics from a dummy module to some base statistics functions

### CODE REVIEW

**Comment by pfalcon** [comment]
Thanks, but: 1) specific commit message please, adhering to https://github.com/micropython/micropython-lib/wiki/ContributorGuidelines; 2) there're code style errors.

File: statistics/statistics.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+def mean(data):
+	average = 0
```
Comment (dpgeorge) [inline statistics/statistics.py]: Please use 4 spaces consistently for indentation level.

File: statistics/statistics.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+def mean(data):
+	average = 0
+	for points in data:
+		average += points
+	return average/len(data)
```
Comment (dpgeorge) [inline statistics/statistics.py]: What if len(data) == 0?

Comment (dpgeorge) [inline statistics/statistics.py]: Why not `average = sum(data)`?

File: statistics/statistics.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+def median(data):
+	"""Return median of sequence data."""
+	sorteddata = sorted(data)
+	dataLen = len(data)
+	index = (dataLen - 1) // 2
+
+	if (dataLen % 2):
+		return sorteddata[index]
+	else:
+		return (sorteddata[index] + sorteddata[index + 1])/2.0
```
Comment (dpgeorge) [inline statistics/statistics.py]: Space around `/` operator.

File: statistics/statistics.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	ss = sum((x-c)**2 for x in data)
+	return ss
+	
+def pstdev(data):
+	"""Calculates the population standard deviation."""
+	n = len(data)
+	if n < 2:
+		raise ValueError('variance requires at least two data points')
+	ss = _ss(data)
+	pvar = ss/n # the population variance
```
Comment (dpgeorge) [inline statistics/statistics.py]: Space around operators.

File: statistics/statistics.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+def variance(data):
+	"""Calculates variance."""
+	if iter(data) is data:
+		data = list(data)
+	n = len(data)
+	if n < 2:
+		raise ValueError('variance requires at least two data points')
+	T, ss = _ss(data)
+	return _convert(ss/(n-1), T)
```
Comment (dpgeorge) [inline statistics/statistics.py]: Is `_convert` defined?

**Comment by dpgeorge** [comment]
Closing due to inactivity.

**Pull request closed without merging**